src/components/Changepersonaldata/Changepersonaldata.css

```diff
@@ -0,0 +1,107 @@
+:root {
+  --first-color: #1a73e8;
+  --input-color: #80868b;
+  --border-color: #dadce0;
+
+  --body-font: "Roboto", sans-serif;
+  --normal-font-size: 1rem;
+  --small-font-size: 0.75rem;
+}
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
+body {
+  margin: 50;
+  padding: 50;
+  font-family: var(--body-font);
+  font-size: var(--normal-font-size);
+}
+h1 {
+  margin: 0;
+}
+
+.l-form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 1000px;
+}
+.form {
+  width: 1000px;
+  padding: 4rem 2rem;
+  border-radius: 1rem;
+  box-shadow: 0 10px 25px rgba(92, 99, 105, 0.2);
+}
+.form__title {
+  font-weight: 400;
+  margin-bottom: 3rem;
+}
+.form__div {
+  position: relative;
+  height: 48px;
+  margin-bottom: 1.5rem;
+}
+.form__input {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
```

Comment:
If you don't need these rules, delete them, cause they cause trouble in the layout ;)

` position: absolute;
  top: 0;
  left: 0;
  width: 100%;`